### PR TITLE
Add more forecast diagnostics

### PR DESCRIPTION
**Pull request by jjguerrette**
### Description
Add more forecast diagnostics, consistent with known MPAS-A verification use-cases

### Issue closed

None

### Tests completed
 - [x] 3dvar_OIE120km_WarmStart
 - [x] 3dvar_OIE120km_ColdStart

### CODE REVIEW

File: config/mpas/forecast/stream_list.atmosphere.diagnostics [hunk 1]
```diff
@@ -10,3 +10,15 @@ cldfrac_tot_UM_max
 cldfrac_tot_UM_rand
 cldht_top_UM
 cldht_base_UM
+height_250hPa
+height_500hPa
```
Comment (jjguerrette) [inline config/mpas/forecast/stream_list.atmosphere.diagnostics]: Per @junmeiban, only `height_500hPa` is needed for 500hPa geopotential height ACC score.  Others are added for possibly useful verification with minimal extra cost.  That avoids the need to rerun forecasts for those variables should we be interested to consider them.

Comment (junmeiban) [reply config/mpas/forecast/stream_list.atmosphere.diagnostics]: Thanks JJ. Would you please add "xtime"? Just as Jake's diag files.

Comment (jjguerrette) [reply config/mpas/forecast/stream_list.atmosphere.diagnostics]: Sorry for missing that.  Should I just copy Jake's entire `stream_list.atmosphere.diagnostics`?

Comment (liujake) [reply config/mpas/forecast/stream_list.atmosphere.diagnostics]: I think it is better you at least include all variables in my stream file, then can add more variables if you like (e.g., z at 250 hPa).

Comment (jjguerrette) [reply config/mpas/forecast/stream_list.atmosphere.diagnostics]: Done.

**Review by junmeiban — APPROVED** [review]

**Pull request merged**